### PR TITLE
Add few more famous brand colors

### PR DESCRIPTION
**Pull request by leozhang2018**
Include [QQ](http://im.qq.com/),[Alibaba](http://www.alibabagroup.com/),[Sina](http://www.sina.com.cn/),[Taobao](https://www.taobao.com/)

### CODE REVIEW

File: data/brandColors.js [hunk 1]
```diff
@@ -757,6 +758,10 @@ var _brandColors = {
   'product-hunt-4': '#00b27f',
   'pusha': '#0f71b4',
   'qantas-airlines': '#e0001b',
+  'qq-1': '#000000',
```
Comment (reimertz) [inline data/brandColors.js]: Just to follow the current (very stupid) semantic that I chose to use, can you please remove the "-1" on the first color? :)

File: data/brandColors.js [hunk 2]
```diff
@@ -836,6 +841,8 @@ var _brandColors = {
   'shopify-4': '#f5f5f5',
   'shopify-5': '#f2f7fa',
   'shopify-6': '#666666',
+  'sina-1': '#e71d34',
```
Comment (reimertz) [inline data/brandColors.js]: Same here.

**Comment by leozhang2018** [comment]
Yeah,i remove it.Please merge new PR.

**Comment by reimertz** [comment]
Thanks bro!

**Pull request merged**